### PR TITLE
Fix/arrow tokens

### LINKED ISSUE #56: Missing Namespaces from Component
## Expected Behavior
It should have some namespaces like ".arrow", ".arrowRight", ".arrowLeft", but those has vanished from carousel component.

## Current Behavior
After last releases, some namespaces have vanished from component, breaking down some styles.

## Possible Solution
I suggest a re-check in last releases to see what is happening, or if this is known by you, update README.md the namespaces removed.

## Steps to Reproduce (for bugs)
1. Clone store-theme into the project folder and reach vtex carousel component trying to see namespaces like ".arrow", "rightArrow" and ".leftArrow"

## Context
The issue has broken down some styles from our stores projects, we just need to understand if this is a bug or a definitive release to work around.

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request? What problem is this solving?
The current tokens of the arrow are different from the old ones for no reason.

#### How should this be manually tested?
[workspace](https://arrowtokens--storecomponents.myvtex.com/)

Also fixes #56 

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/styles.css [hunk 1]
```diff
@@ -11,3 +11,6 @@
   height: 10px;
   width: 10px;
 }
+
+.arrowLeft {}
```
Comment (arthurepc) [inline react/styles.css]: Is this necessary?

File: react/styles.css [hunk 2]
```diff
@@ -11,3 +11,6 @@
...
+.arrowRight {}
```
Comment (arthurepc) [inline react/styles.css]: same

File: react/styles.css (hunk 1 above)
Comment (JuanAlmeida12) [reply react/styles.css]: Yes, we keep the CSS handles to avoid undefined classes in DOM.

Comment (Klynger) [reply react/styles.css]: Yeah, if you don't put it will put a undefined class to the element

**Pull request merged**